README.md

```diff
@@ -94,4 +94,4 @@ But the complete set of 50+ is shown in the [User Guide](http://colorcet.holoviz
 ## Colorcet, HoloViz, and PyViz
 
 Colorcet is part of the [HoloViz](https://holoviz.org) family of tools. The [holoviz.org](https://holoviz.org) website shows how to use Colorcet together with other libraries to solve complex problems, with detailed tutorials and examples. Example projects using Colorcet are at
-[examples.pyviz.org](https://examples.pyviz.org), and you can compare Colorcet to other available tools at [pyviz.org](https://pyviz.org).
\ No newline at end of file
+[examples.pyviz.org](https://examples.pyviz.org), and you can compare Colorcet to other available tools at [pyviz.org](https://pyviz.org).
```

Comment:
I made the edit on line 11 and the PR via the github ui...no idea what happened here on line 97.

Reply:
I guess it's a newline change, but there's no visual indication (at least that I can see) of whether it's adding or removing...

Reply:
I checked locally - it's adding one, so that's ok :)